step-agent/pom.xml

```diff
@@ -51,7 +49,7 @@
 								<artifactItem>
 									<groupId>ch.exense.step</groupId>
 									<artifactId>step-grid-agent</artifactId>
-									<version>${step-grid.version}</version>
+<!--									<version>${step-grid.version}</version>-->

```

Comment:
I assume this is not required anymore or the build would not go trough, the comment should be removed in that case.